### ISSUE TITLE
Duplicated icon render

### ISSUE THREAD

**Issue by mohamedsabil83** [laravolt/semantic-form#5]
When use **->prependIcon($icon)** or **->appendIcon($icon)**, the Icon appear once but in code it's rendered twice.
e.g.
```html
<div class="field">
  <label>Username</label>
  <div class="ui input icon">
    <input type="text" name="username">
    <i class="icon user"></i><i class="icon user"></i>
  </div>
</div>
```

**Comment by uyab** [comment]
Can you show the all the view code  in here?

**Comment by mohamedsabil83** [comment]
**View**:
```php
{!! SemanticForm::post(route('login')) !!}

    {!! SemanticForm::input('identity_no')->label(trans('validation.attributes.identity_no'))
     ->prependIcon('payment')
     ->hint($errors->first('identity_no'), $errors->has('identity_no') ? 'ui pointing red basic label' : '') !!}

    {!! SemanticForm::input('password')->label(trans('validation.attributes.password'))
     ->prependIcon('lock')->type('password') !!}

    {!! SemanticForm::submit(lang('main.button.login'))->clear('class')->addClass('uk-button uk-button-primary') !!}

    {!! SemanticForm::close() !!}
```

**Render**:
```html
<form method="POST" action="http://127.0.0.1:8000/login" class="ui form"><input type="hidden" name="_token" value="DnBSRsd9kzHExTJjDodCDtF5Awx1Vnuc7r2qOksj">

    <div class="field"><label>Identity No.</label><div class="ui input left icon"><i class="icon payment"></i><i class="icon payment"></i><input type="text" name="identity_no"></div><div class="hint"></div></div>

    <div class="field"><label>Password</label><div class="ui input left icon"><i class="icon lock"></i><i class="icon lock"></i><input type="password" name="password"></div></div>

    <button type="submit" class="uk-button uk-button-primary">Login</button>

    </form>
```

**Comment by uyab** [comment]
Fixed in 1.10.3
